src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Slowest average response time
+          </td>
+
+          <td>
+            The slowest transactions to complete. A transaction could be called only a few times, but if it is one of the longest transactions to finish, it would appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Apdex most dissatisfying
+          </td>
+
+          <td>
+            The transactions with the highest percentages of user dissatisfaction levels for the response time of your web apps and services ([Apdex](/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction)).
+
+            The [dissatisfaction level](/docs/apm/new-relic-apm/apdex/view-your-apdex-score#apdex-dissat) is the difference between a perfect Apdex score (1.0) and your app's `apdex_t` ("tolerating") score, based on your [Apdex settings](/docs/apm/new-relic-apm/apdex/change-your-apdex-settings). Transactions that have the highest (worst) percentages appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Throughput (requests per minute)
+          </td>
+
+          <td>
+            The amount of [throughput](/docs/using-new-relic/welcome-new-relic/getting-started/glossary#throughput) in operation requests per minute (RPM). Transactions that have the highest RPM appear at the top of this ranking.
+          </td>
+        </tr>
+
+      </tbody>
+    </table>
+  </Collapser>
+
+  <Collapser
+    id="chart-views"
+    title="View transaction data in different visual formats"
+  > 
+    ![Screenshot showing selectors to change the chart.](./images/chart_type_selectors.png "chart_type_selectors.png")
+
+    The **Transaction** page's **Top web transactions** and **Top non-web transactions** charts, as well the **App performance** window for a selected transaction include options to view data as a [chart](/docs/using-new-relic/user-interface-functions/view-your-data/select-chart-views), [histogram](/docs/using-new-relic/user-interface-functions/view-your-data/histograms-view-data-distribution), or [percentile](/docs/using-new-relic/user-interface-functions/view-your-data/percentiles-compare-ranked-data). 
+
+    In addition, click **Breakdown** table for a selected transaction to show or hide all segments of the breakdown data.
+  </Collapser>
+
+  <Collapser
+    id="browser-data"
+    title="Show corresponding browser request data"
+  >
+    If you haven't already, [install the browser agent](/docs/new-relic-browser/browser-settings) to view browser information. 
+    
+    Then, to view corresponding [browser request](/docs/browser/new-relic-browser/additional-standard-features/page-views-understanding-your-sites-popularity) data, either click **Browser drill-down** or go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Page views**.
```

Comment:
```suggestion
    Then, to view corresponding [browser request](/docs/browser/new-relic-browser/additional-standard-features/page-views-understanding-your-sites-popularity) data, go to **[one.newrelic.com](https://one.newrelic.com) > Browser > (select an app) > Page views**.
```

We will need to add back the link from a transaction to its corresponding Browser page view, though it's not in scope at the moment. FYI @goldenplec